src/SampleLibraryUI/Examples/DropDown.cs

```diff
@@ -13,6 +14,9 @@ public class DropDownExample : DSDropDownBase
     {
         public DropDownExample() : base("item"){}
 
+        [JsonConstructor]
+        public DropDownExample(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base("item") { }
```

Comment:
might be useful to explain you are passing the deserialized ports with the json constructor and then calling the base class... also... is this correct? you don't also need to pass the ports to the base class?

Reply:
@mjkkirschner nice catch! should be 
```c# 
public DropDownExample(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base("item", inPorts, outPorts) { }
```